flexget/plugins/input/anidb_feed.py

```diff
@@ -0,0 +1,453 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+import re
+import feedparser
+from xml.dom import minidom
+from time import mktime
+from datetime import datetime
+
+from requests import RequestException
+
+from flexget import plugin
+from flexget.config_schema import one_or_more
+from flexget.event import event
+from flexget.entry import Entry
+from flexget.utils.qualities import Quality
+from flexget.utils.tools import str_to_boolean, str_to_naive_utc, str_to_number
+from flexget.utils.cached_input import cached
+from flexget.utils.requests import TimedLimiter
+
+
+log = logging.getLogger('anidb_feed')
+
+HEADER = {'User-agent': 'Mozilla/5.0'}  # anidb needs valid header or we get '403 Client Error'
+LIMITER = TimedLimiter('anidb.net', '2 seconds')  # default anidb api limit
+
+
+NAMESPACE_NAME = 'xhtml'
+NAMESPACE_URL = 'http://www.w3.org/1999/xhtml'
+NAMESPACE_TAGNAME = 'dl'
+NAMESPACE_PREFIX_MAIN = 'anidb_'
+NAMESPACE_PREFIX = 'anidb_feed_'
```

Comment:
I'm not a fan of this naming convention. Namespace is very C++/C#-esque. At a glance, it seems like you're using prefix and prefix_main as prefixes for entry fields. I'd rather the variables were called something like `field_prefix`.